build/commands/lib/config.js

```diff
@@ -24,10 +24,11 @@ const run = (cmd, args = []) => {
 }
 
 // this is a huge hack because the npm config doesn't get passed through from brave-browser .npmrc/package.json
-var packageConfig = function(key){
+var packageConfig = function(key, sourceDir = rootDir){
   let packages = { config: {}}
-  if (fs.existsSync(path.join(rootDir, 'package.json'))) {
-    packages = require(path.relative(__dirname, path.join(rootDir, 'package.json')))
+  const configAbsolutePath = path.join(sourceDir, 'package.json')
```

Comment:
can we not call this `sourceDir`, it had me confused for a while because I thought it was `srcDir`. Maybe just `dir` or something?

Reply:
`npmProjectDir` perhaps

Reply:
is this change still pending?

Reply:
no, it's done